src/clj/ataru/tarjonta_service/hakuaika.clj

```diff
@@ -20,6 +20,9 @@
       current-hakuaika
       (last hakuaikas))))
 
+(defn- jatkuva-haku? [haku]
+  (clojure.string/starts-with? (:hakutapaUri haku) "hakutapa_03"))
```

Comment:
Is the `starts-with?` good enough here? Just thinking if there are/will be cases with `hakutapa_031` etc possible...